### ISSUE TITLE
request to add support for editing document metadata

### ISSUE THREAD

**Issue by stanley-ibm** [sysalto/ReactiveReports#9]
Could there be any support for adding document metadata, such as title, author, subject, keywords?

**Comment by sysalto** [comment]
Yes, these can be done, but they are more work-intensive than the previous changes. These would take around 4 months to do.

**Comment by MarcusChienIBM** [comment]
Is the work for this complete? If so, is there a reason that this issue has not been closed yet?
